sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/eventsourcedentity/EventSourcedEntitiesTestModels.java

```diff
@@ -74,6 +78,31 @@ private Integer privateMethodSimilarSignature(Integer event) {
     }
   }
 
+  @Entity(entityKey = "id", entityType = "counter")
+  @RequestMapping("/eventsourced/{id}")
+  public static class WellAnnotatedESEntityWithJWT extends EventSourcedEntity<Integer> {
+
+    @GetMapping("/int/{number}")
+    @Subscribe.ValueEntity(UserEntity.class)
```

Comment:
```suggestion
```

That's an entity subscribing to another entity. That's not something I think we support. 
Maybe a copy-paste from some other test? 

In any case, I think there is another issue here. This methods are not even returning an Effect and we are just processing it anyway. 

We may want to support raw types later on, but I'm surprised that we are not getting an error here.